source/nodejs/README.md

```diff
@@ -10,6 +10,7 @@ This directory contains all the JS packages for adaptive cards:
 * `adaptivecards-designer-app` - The designer app that consumes the designer component
 * `adaptivecards-site` - The generated website that is currently hosted at adaptivecards.io
 * `adaptivecards-visualizer` - The classic card Visualizer
+* `react-adaptivecards` - React.js library
```

Comment:
What was the reason behind the decision to break from the `adaptivecards-<name>` convention?

Reply:
Also, there looks to be an existing package with the `react-adaptivecards` name

Reply:
@matthidinger I'm calling you in on this one

Reply:
Isn't "react-" the React convention?

Reply:
I believe @matthidinger plans to reach out to the existing package maintainers as the current one isn't actively developed any longer.